src/syft/lib/torchvision/allowlist.py

```diff
@@ -320,162 +190,72 @@
-    "return_type": "torch.Tensor",
-    "min_version": "0.8.0",
-    "test_parameters": "(tens, 10 , 20, 30, 40)",
-    # Torch 1.6 expects input to be PIL image, so minimum version as 0.7 (Torch 1.7.0)
-}
+allowlist["torchvision.transforms.functional.crop"] = "torch.Tensor"
 allowlist["torchvision.transforms.functional.equalize"] = {
     "return_type": "torch.Tensor",
     "min_version": "0.9.0",
-    "test_parameters": "(tens)",
-}
-allowlist["torchvision.transforms.functional.erase"] = {
-    "return_type": "torch.Tensor",
-    "test_parameters": "(tens, 10, 20, 30, 40, 250)",
-    "min_version": "0.8.0"
-    # Torch 1.6 expects input to be PIL image, so minimum version as 0.7 (Torch 1.7.0)
-}
-allowlist["torchvision.transforms.functional.five_crop"] = {
-    "return_type": "torch.Tensor",
-    "test_parameters": "(tens, 10)",
-    "min_version": "0.8.0"
-    # Torch 1.6 expects input to be PIL image, so minimum version as 0.7 (Torch 1.7.0)
 }
+allowlist["torchvision.transforms.functional.erase"] = "torch.Tensor"
+allowlist["torchvision.transforms.functional.five_crop"] = "torch.Tensor"
 allowlist["torchvision.transforms.functional.gaussian_blur"] = {
     "return_type": "torch.Tensor",
     "min_version": "0.8.0",
-    "test_parameters": "(tens, 3)",
 }
 allowlist["torchvision.transforms.functional.hflip"] = "torch.Tensor"
 allowlist["torchvision.transforms.functional.invert"] = {
     "return_type": "torch.Tensor",
     "min_version": "0.9.0",
-    "test_parameters": "(tens)",
-}
-allowlist["torchvision.transforms.functional.normalize"] = {
-    "return_type": "torch.Tensor",
-    # "test_parameters": "(tens, [0.5, 0.5, 0.5], [1, 1, 1]).unsqueeze(0)",
-    # currently commenting because of test issues with hier versions
-    # (//) works for 1.6.0 and / works for higher version :(
 }
+allowlist["torchvision.transforms.functional.normalize"] = "torch.Tensor"
+# "test_parameters (tests/lib/torchvision/allowlist_test_parameters.json): (tens, [0.5,
+#  0.5, 0.5], [1, 1, 1]).unsqueeze(0)",
+# currently not added because of test issues with hier versions
+# (//) works for 1.6.0 and / works for higher version :(
+
 allowlist["torchvision.transforms.functional.pad"] = {
     "return_type": "torch.Tensor",
-    "test_parameters": "(tens, 10)",
-    "min_version": "0.8.0"
-    # # Torch 1.6 expects input to be PIL image, so commenting this currently
-}
-allowlist["torchvision.transforms.functional.perspective"] = {
-    "return_type": "torch.Tensor",
-    "test_parameters": "(tens, [[10,20],[20,30],[30, 40],[40,50]], [[20,30],[30,40],[40, 50],[50,60]])",
-    "min_version": "0.8.0"
-    # # Torch 1.6 expects input to be PIL image, so commenting this currently
+    "min_version": "0.6.0",
 }
+allowlist["torchvision.transforms.functional.perspective"] = "torch.Tensor"
 
 # Converts PIL to tensor, currently not supported
 # allowlist["torchvision.transforms.functional.pil_to_tensor"] = "torch.Tensor"
```

Comment:
We have PIL image types now so this could potentially be enabled?

Reply:
Yes, done.